src/lib/pokemonApi.ts

```
@@ -1,4 +1,6 @@
-import { PokemonCard, CardPrices } from './types';
+import { PokemonCard, CardPrices, CardSet } from './types';
```

ESLint

[error] 'CardSet' is defined but never used.

(@typescript-eslint/no-unused-vars)